PROTOCOL.md

```diff
@@ -351,6 +351,28 @@ animation) and `|[silent]` (suppress message).
 > turn. Otherwise, it means that the weather has changed due to a move or ability,
 > or has expired, in which case `WEATHER` will be `none`.
 
+`|-fieldstart|MOVE`
+
+> Field condition `MOVE` started. This includes all moves that effect the entire
+> field but isn't a weather such as Trick Room or any Terrain move.
+
+`|-fieldend|MOVE`
+
+> Indicates that the condition `MOVE` ended.
+
+`|-sidestart|SIDE|MOVE`
+
+> A side condition `MOVE`, such as Entry Hazards, Reflect/Light Screen/Safeguard,
+> or Tailwind, is started on `SIDE`. 
+
+`|-sideend|SIDE|ACTION`
+
+> Indicated when a side condition ends on `SIDE`. For conditions like Tailwind that
+> end by themselves, `ACTION` will only specify which condition that ended. If the
+> side condition was forcefully removed `ACTION` will be `COND|FROM|OF`, where `COND`
+> is the removed side condition, `FROM` is the cause of removal and `OF` is who
+> caused the removal.
```

Comment:
Don't mention `[from]` and `[of]` -- the top of the Minor Actions section already mentions them.